react/utils/index.js

```diff
@@ -0,0 +1,73 @@
+const MAX_RETRIES = 3
```

Comment:
A more meaningful name for utils files. Put functions that aren't related in separated files.

Reply:
If i separate those functions in different files, I would have a file with 3 lines and only one function, the translate function.

Reply:
I don't see a problem with this.